edb/server/pgsql/ast.py

```diff
@@ -336,6 +337,10 @@ class DML(Base):
     # List of expressions returned
     returning_list: typing.List[ResTarget]
 
+    @property
+    def target_list(self):
+        return self.returning_list
```

Comment:
Maybe it's better to define an `is_nullable()` method?

Reply:
This is not related to nullability. This is to avoid having to do `if isinstance(DML): returning_list.append(...) else target_list.append(...)`

Reply:
It's weird that a base class makes `target_list` and `returning_list` the same, even though they have distinctly different names with a different meaning, no?